### PR TITLE
KAFKA-1621 : Standardize --messages option

### PR DESCRIPTION
**Pull request by rekhajoshm**
KAFKA-1621: Standardize --messages option in perf scripts

### CODE REVIEW

File: core/src/main/scala/kafka/tools/ConsumerPerformance.scala [hunk 1]
```diff
@@ -120,7 +118,7 @@ object ConsumerPerformance {
 
     val options = parser.parse(args: _*)
 
-    CommandLineUtils.checkRequiredArgs(parser, options, topicOpt, zkConnectOpt)
+    CommandLineUtils.checkRequiredArgs(parser, options, topicOpt, zkConnectOpt,numMessagesOpt)
```
Comment (nehanarkhede) [inline core/src/main/scala/kafka/tools/ConsumerPerformance.scala:121]: space after comma

File: core/src/main/scala/kafka/tools/SimpleConsumerPerformance.scala [hunk 2]
```diff
@@ -141,7 +143,7 @@ object SimpleConsumerPerformance {
 
     val options = parser.parse(args : _*)
 
-    CommandLineUtils.checkRequiredArgs(parser, options, topicOpt, urlOpt)
+    CommandLineUtils.checkRequiredArgs(parser, options, topicOpt, urlOpt, numMessagesOpt)
```
Comment (nehanarkhede) [inline core/src/main/scala/kafka/tools/SimpleConsumerPerformance.scala:146]: same here?

File: core/src/main/scala/kafka/tools/SimpleConsumerPerformance.scala [hunk 3]
```diff
@@ -31,10 +31,12 @@ import kafka.common.TopicAndPartition
...
 object SimpleConsumerPerformance {
 
+  private val logger = Logger.getLogger(getClass())
+
   def main(args: Array[String]) {
-    val logger = Logger.getLogger(getClass)
-    val config = new ConsumerPerfConfig(args)
 
+    val config = new ConsumerPerfConfig(args)
+    logger.info("Starting simpleconsumer...")
```
Comment (nehanarkhede) [inline core/src/main/scala/kafka/tools/SimpleConsumerPerformance.scala:39]: simpleconsumer => SimpleConsumer

**Comment by rekhajoshm** [comment]
Thanks @nehanarkhede Applied your review comments at #58 Closing this.Thanks

**Comment by ijuma** [comment]
@rekhajoshm, this is still open. Can you please close it?

**Pull request closed without merging**